reviews/2020-strimzi-annual.md

```diff
@@ -0,0 +1,73 @@
+* The [#strimzi Slack channel](https://cloud-native.slack.com/archives/CMH3Q3SNP) now has over 550 members who create a vibrant community with many questions and discussions
+
+Despite the the number of new contributors, most of the contributions are still carried by the core contributors from Red Hat. We would like to have more people contribute regularly to the project and not just occasionally with one-of PRs.
+
+All Strimzi GitHub metrics can be found in the [CNCF DevStats dashboards](https://strimzi.devstats.cncf.io/d/8/dashboards).
+
+## Maintainers
+
+Strimzi has currently 5 maintainers from two companies (Red Hat and IBM). Two new maintainers were added since joining CNCF. The full maintainer list can be found in our [GitHub repository](https://github.com/strimzi/governance/blob/master/MAINTAINERS).
+
+Additionally to the maintainers, 3 other contributors have _Write_ permissions to individual repositories and act as owners for the sub-projects within these repositories.
+
+## Adoption
+
+Strimzi project sees growth in adoption both in terms of production users as well as in terms of companies engaging with the project via issues, PRs or Slack channels. The list of our adopters can be found in our [GitHub repository](https://github.com/strimzi/strimzi-kafka-operator/blob/master/ADOPTERS.md) and includes end-users such as:
+
+* [Axual](https://axual.com/)
+* [Grupo MASMOVIL](https://www.grupomasmovil.com/)
+* [Helvetia](https://helvetia.com/)
+* [Marlow Navigation](https://marlow-navigation.com/)
+* [SBB CFF FFS](https://www.sbb.ch/en/home.html)
+
+Strimzi is also adopted by several vendors and used in their products:
+* [IBM](https://www.ibm.com/cloud/event-streams)
+* [Lightbend](https://cloudflow.io/)
+* [Red Hat](https://www.redhat.com/en/resources/amq-streams-datasheet)
+
+_(While Red Hat is owned by IBM, Strimzi is currently used in two separate product offerings - one from IBM and one from Red Hat)_
+
+## Past Goals
+
+This is our first annual review, so there are no recorded goals from the previous one. But our general goals and expectations when joining CNCF Sandbox were:
+* Attract more contributors and grow the community thanks to the independent governance of CNCF
+* Attract more users through the increased visibility
+
+These goals are being fulfilled as seen from the new users and new contributors statistics.
+
+## Current Goals
+
+For the next year, we would like to achieve the following goals:
+* Keep attracting more users to Strimzi and Apache Kafka on Kubernetes
+* Keep broadening the contributor base with focus on trying to get also more regular and long-term committers. This should also lead to adding more maintainers from other companies than just IBM and Red Hat.
+* Release Strimzi 1.0. While we see Strimzi as production-ready and have production users, we are still on `0.x` versions. We should move to 1.0 during the next year.
```

Comment:
Can we include the link to the list of features/changes planned for 1.0 release?

Reply:
And can you also include the link to the project Roadmap

Reply:
I added a list of the main features we would like to have for 1.0 release as well as link to our high-level roadmap.